docs/proposals/Popover/README.md

```diff
@@ -0,0 +1,131 @@
+			>
+				{renderProps => (
+					{/* Propover contents */}
+					<Card>
+						<strong>
+							Attachments are excluded by default now!
+						</strong>
+						<button onClick={renderProps.dissmiss}>
+							Okay!
+						</button>
+					</Card>
+				)}
+			</Popover>
+		</>
+	)
+}
+```
+
+```tsx
+// OutOfTheBox.tsx
+import { Popover } from '...'}
+
+const OutOfTheBox = () => {
+    return (
+        <PopoverBase
+            attachTo={<button>No scrubs</button>}
+            activateOn={'focus'}
+        >
+            <marquee>Party like it's 1999</marquee>
+        <PopoverBase>
+    )
+}
+
+```
+
+## Props Table
+
+### Popover
+
+| name        | type                                                                           | required? | default   | description                                                                                                                                                                                      |
+| ----------- | ------------------------------------------------------------------------------ | --------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| attachTo    | `HTMLElement \| React.Ref<HTMLElement \| null> \| React.ReactNode `            | ✓         |           | Element the popover will attach to and point at.                                                                                                                                                 |
+| activateOn  | `"click" \| "focus"`                                                           |           | `"click"` | When `attachTo` is React node it it will be cloned and the popover will be toggled _on click_ by default.                                                                                        |
+| dismissable | `boolean`                                                                      |           | `true`    | Whether or not the popover is dismissable.                                                                                                                                                       |
+| toggled     | `boolean`                                                                      |           |           | Control popover viability rather than letting the component take care of it.                                                                                                                     |
+| children    | `React.ReactNode \| (renderProps: { dismiss: () => void }) => React.ReactNode` | ✓         |           | Pop-over content. A render function may also be provided and will provide you with a callback to dismiss the popover and potentially other stateful information or access to popper related APIs |
+| arrow       | `PopperJS.Placement`                                                           |           | `"auto"`  |
```

Comment:
Do we have a case yet where we need to specify the arrow direction? Wondering if we can just set this to auto for now and think about it more in the future if we have a case that really needs it.